cves/CVE-2011-3192.yml

```diff
@@ -32,7 +32,11 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+    This vulnerability involves resource exhaustion leading to a denial of service
+    due to improper handling of overlapping byte ranges in http requests. The eventual fix nearly 7 years later involved reducing the 
```

Comment:
**Grammar**
The eventual fix`, nearly 7 years later,` ...

Reply:
I think I've gotten too used to writing with Grammarly at my side. Thanks for catching!